pubspec.yaml

```diff
@@ -15,7 +15,7 @@ dependencies:
   drift: ^2.18.0
```

Comment:
@alexdivadi  please make a pull request and/or jira ticket to get rid of the carets in pubspec.yaml so we see prs like this for every pubspec.lock change.